src/song_utils.py

```diff
@@ -84,26 +82,44 @@ def song_format(
     # Sanitize tag contents.
     # We explicitly check for None here, as anything else means that the data was
     # pulled from the audio.
-    if artist is not None:
-        artist = sanitize_tag(artist)
-    if title is not None:
-        title = sanitize_tag(title)
-
-    # Display in the format <Artist-tag> - <Title-tag>
-    # If no artist tag use fallback if valid. Otherwise, skip artist
+    if not artist:
```

Comment:
I don't think we meant to change from `if artist is not None` to `if not artist`, which is different for the case of `artist = ""`.

Reply:
It's not so much that we changed X line to Y and more that we reorganized the entire logic and the first two lines happened to look similar.

In the first version `if artist is not None` was just gating the sanitization of the tag read from the mp3. If it was "" then it would be sanitized and nothing would change. Then a layer `if artist` statement would choose whether the artist was put in the final string

This is closer to a replacement for the `else` clause of the `if artist` later in the original. As it gates whether or not the fallback is used, not sanitization. Then the later `if artist` is the new sanitization gate.

Reply:
Right, I hadn't realised this was more than just a typo. Thanks for the detailed explanation!